content/_code-samples/checks/js/cancel_check.js

```diff
@@ -0,0 +1,44 @@
+const xrpl = require('xrpl');
+
+const secret = "sEdTPPEeMH6SAgpo6rSj8YW7a9vFfUj"; // TODO: Replace with your secret
+const checkId = ""; // TODO: Replace with your check ID
+
+const main = async ()=> {
+  try {
+      
+    // Connect ----------------------------------------------------------------
+    const client = new xrpl.Client('wss://s.altnet.rippletest.net:51233');
+    await client.connect();
+
+    // Check if the check ID is provided --------------------------------------
+    if (checkId.length === 0) {
+      console.log("Please provide a check ID");
```

Comment:
It's a bit vague on how someone using this snippet could provide the check id.
```suggestion
      console.log("Please edit this snippet to provide a check ID. You can get a check ID by running create_check.js.");
```

Reply:
(Applies to both this and `cash_check.js`)

Reply:
done